### PR TITLE
Api login with google

### PR DESCRIPTION
**Pull request by aAnurag7**

### CODE REVIEW

File: server/DB/connection.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+const mongoose = require('mongoose');
+const dotenv = require('dotenv')
+dotenv.config({path:'../config.env'})
+const DB = "mongodb+srv://anurag:anuragchatur@cluster0.r9fqxa8.mongodb.net/?retryWrites=true&w=majority"
```
Comment (UrAvgDeveloper) [inline server/DB/connection.js:4]: does this url contain any critical info?? should it be in .env??

File: server/config.env [hunk 2]
```diff
@@ -0,0 +1,2 @@
+DATABASE="mongodb+srv://anurag:anuragchatur@cluster0.r9fqxa8.mongodb.net/?retryWrites=true&w=majority"
+SECRET_KEY = MDSFHSKAHFDJKBAKJSBSAJKBFJSDKABJFASBFKASFASKBAFFDSGSDGSDGSD
```
Comment (UrAvgDeveloper) [inline server/config.env:2]: don't push .env ever!!!!!

**Review by UrAvgDeveloper — COMMENTED** [review]
thanks for pr, please take care of spacing and indentation

File: server/config.env (hunk 2 above)
Comment (aAnurag7) [reply server/config.env]: resolve in <[e9d48ff]>(https://github.com/aAnurag7/todoList/pull/7/commits/e9d48ff8472e2ad7936acbbb195981037bf99d10)

File: server/DB/connection.js (hunk 1 above)
Comment (aAnurag7) [reply server/DB/connection.js]: resolve in <[5091cd2]>(https://github.com/aAnurag7/todoList/pull/7/commits/5091cd2d94ebd8965ec87fc353b98f04d12287a9)

**Review by UrAvgDeveloper — APPROVED** [review]

**Pull request merged**